requirements.txt

```
@@ -1 +1,2 @@
 pyhf[backends]~=0.4
+wandb==0.9.1
```

Given that they're at minor release version `0.9` they probably have a mature/stable enough API that you don't need to pin all the way down to the patch release. Can probably just use
```suggestion
wandb~=0.9
```

---

Updated!